packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -166,16 +166,34 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 		const changeTimeStamp = new Date(changeEntry.timestamp);
 		const wasChangeToday = nowTimeStamp.getDate() === changeTimeStamp.getDate();
 
+        const accordianBackgroundColor: IStackItemStyles = {
+            root: {
+                background: changeEntry.changeKind === "added" ? "lightgreen" : "orange",
```

Comment:
100% just my own style opinions (which are questionable 😋) would light-red be a better removed color? (Feel free to ignore this 🙂)

Reply:
`background: changeEntry.changeKind === "added" ? "#90ee90" : "#ed1d24",`

Changed colors with RGB-code, light-green and light-red for `added` and `removed` respectively (as shown in picture above)

Reply:
I hate to say it, but I think that red might be too dark. Something lighter would probably be better (to make the black text easier to read). Maybe something like `#FF7377`? 

Again, I'm not a designer, so take my feedback with a grain of salt, and do what you think works best :)

Reply:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/111468570/206538945-193ab569-15ab-4230-b11a-816f72303d17.png">

`#FF7377` applied 😄. I'm not a good designer, and I even a green/red color blinded. So, I will take your suggestion!

Reply:
I didn't know that! I think your thoughts here are particularly useful then, as we want this tooling to be accessible! Did one of the other color options work better for you?